lm_eval/models/gpt2.py

```diff
@@ -60,8 +60,8 @@ def __init__(
                         trust_remote_code=trust_remote_code,
                         )
 
-                
-        else:
+        elif isinstance(pretrained, str):
```

Comment:
Something I realized last minute, bit cleaner than using an assertion. The else block raises a `TypeError`, which is more descriptive.